packages/backend/src/peripherals/database/PositionUpdateRepository.ts

```diff
@@ -25,12 +25,16 @@ export class PositionUpdateRepository
   }
 
   async addOrUpdate(records: PositionUpdateRecord[]) {
+    if (records.length === 0)
+      return this.logger.debug({ method: 'addOrUpdate', rows: 0 })
```

Comment:
This couples the return type of logger.debug to this function. I'd prefer the longer option, when return is written on a new line.